frontend/src/communications/userApi.js

```diff
@@ -14,6 +14,17 @@ export const login = (value) => {
   return api.post('/login', value)
 }
 
+export const forgottenpassword = async (value) => {
+  try {
+    const response = await api.post('/forgottenpassword', value)
+    return response
+  } catch (e) {
+    if (e.response.status === 400) throw new Error('no user')
+    if (e.response.status === 404 || e.response.status === 500)
```

Comment:
ez lehetne inkább else nem? tehát valahogy így
`if (e.response.status === 400) {
  throw new Error('no user')
} else if (e.response.status === 404 || e.response.status === 500) {
  throw new Error('no server')
} else {
  throw new Error(e) // ezzel ugye minden egyéb error is tovább van dobva...
}`

Így minden egyéb hibára ami valaha bekövetkezhet le lesz kezelve... esetleg magát az errort is tovább lehet dobni az első 2 esetben is